packages/common/pipes/file/file-type.validator.ts

```diff
@@ -0,0 +1,17 @@
+import { FileValidator } from './file-validator.interface';
+
+export type FileTypeValidatorOptions = {
+  fileType: string;
+};
+
+export class FileTypeValidator extends FileValidator<FileTypeValidatorOptions> {
+  buildErrorMessage(): string {
+    return `Validation failed (expected type is ${this.validationOptions.fileType})`;
+  }
+
+  isValid(file: any): boolean {
+    if (!this.validationOptions) return true;
```

Comment:
```suggestion
    if (!this.validationOptions) {
       return true;
    }
```
just for the sake of consistency